region_migrations/watch.go

```diff
@@ -0,0 +1,80 @@
+package region_migrations
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/Scalingo/cli/config"
+	scalingo "github.com/Scalingo/go-scalingo"
+	"github.com/fatih/color"
+	errgo "gopkg.in/errgo.v1"
+)
+
+const (
+	SUCCESS = "✔"
+	ERROR   = "✘"
+)
+
+func WatchMigration(client *scalingo.Client, appId, id string) error {
+	refresher := NewRefresher(client, appId, id)
+	migration, err := refresher.Start()
+	if err != nil {
+		return errgo.Notef(err, "fail to watch migration")
+	}
+
+	if migration == nil {
+		return nil
+	}
+
+	if migration.Status == scalingo.RegionMigrationStatusDone {
+		newRegionClient, err := config.ScalingoClientForRegion(migration.Destination)
+		if err != nil {
+			color.Green("The application has been migrated!\n")
```

Comment:
I don't really understand why a successful message here whereas it's an error case. Same question for the following error case.

Reply:
OK nice. Maybe add a comment after the line `if migration.Status == scalingo.RegionMigrationStatusDone {` to explain that all the error case are ignored for this reason?